yarn-project/noir-contracts/src/contracts/docs_example_contract/src/options.nr

```diff
@@ -4,35 +4,19 @@ use dep::aztec::note::note_getter_options::{NoteGetterOptions, Sort, SortOrder};
 use dep::std::option::Option;
 
 // docs:start:state_vars-NoteGetterOptionsSelectSortOffset
-pub fn create_account_card_getter_options(
-    account_address: Field,
-    offset: u32,
-) -> NoteGetterOptions<CardNote, CARD_NOTE_LEN, Field> {
-    NoteGetterOptions::new()
-        .select(2, account_address)
-        .sort(0, SortOrder.DESC)
-        .set_offset(offset)
+pub fn create_account_card_getter_options(account_address: Field, offset: u32) -> NoteGetterOptions<CardNote, CARD_NOTE_LEN, Field> {
+    NoteGetterOptions::new().select(2, account_address).sort(0, SortOrder.DESC).set_offset(offset)
 }
 // docs:end:state_vars-NoteGetterOptionsSelectSortOffset
 
 // docs:start:state_vars-NoteGetterOptionsMultiSelects
-pub fn create_exact_card_getter_options(
-    points: u8,
-    secret: Field,
-    account_address: Field,
-) -> NoteGetterOptions<CardNote, CARD_NOTE_LEN, Field> {
-    NoteGetterOptions::new()
-        .select(0, points as Field)
-        .select(1, secret)
-        .select(2, account_address)
+pub fn create_exact_card_getter_options(points: u8, secret: Field, account_address: Field) -> NoteGetterOptions<CardNote, CARD_NOTE_LEN, Field> {
```

Comment:
@f01dab1e Not wrapping function declaration looks like the only big issue with the formatter now. Everything else are minor issues.

Reply:
If needed, we support ignoring elements through `// noir-fmt:ignore`

Reply:
But it seems in this case we have a bug, as we are not taking into account the size of the function name